apps/emqx_resource/src/emqx_resource_buffer_worker.erl

```diff
@@ -73,9 +73,8 @@
 -type id() :: binary().
 -type index() :: pos_integer().
 -type expire_at() :: infinity | integer().
--type queue_query() :: ?QUERY(from(), request(), HasBeenSent :: boolean(), expire_at()).
+-type queue_query() :: ?QUERY(reply_fun(), request(), HasBeenSent :: boolean(), expire_at()).
```

Comment:
The first argument should still be `from()`, right?  For example, when performing a sync request.